packages/core/src/sass/app/_common/layout/app-layouts.scss

```diff
@@ -146,9 +146,17 @@
             height: calc(100% - #{$MOBILE_HEADER_DTRADER_V2_HEIGHT});
             margin-block-start: $MOBILE_HEADER_DTRADER_V2_HEIGHT;
 
+            &:has(.trade-notification) {
+                position: unset;
```

Comment:
This container (with className _app-contents--is-dtrader-v2_) have only 2 children: _.bottom-nav_ (which have no positioning) and _.trade-notification_ (which appear in the DOM, only when the contract was purchased/closed; has absolute positioning). So, when I removed positioning from _app-contents--is-dtrader-v2_, it should affect only notifications and that is exactly what I need. 
Changes will be tested later as a part trading page testing or as a part of notifications testing